reliability/single-fit/rum/src/test/kotlin/com/datadog/android/rum/integration/ManualTrackingRumTest.kt

```diff
@@ -0,0 +1,407 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.rum.integration
+
+import com.datadog.android.api.feature.Feature
+import com.datadog.android.core.stub.StubSDKCore
+import com.datadog.android.rum.GlobalRumMonitor
+import com.datadog.android.rum.Rum
+import com.datadog.android.rum.RumActionType
+import com.datadog.android.rum.RumConfiguration
+import com.datadog.android.rum.RumErrorSource
+import com.datadog.android.rum.RumResourceKind
+import com.datadog.android.rum.RumResourceMethod
+import com.datadog.android.rum.integration.tests.assertj.hasRumEvent
+import com.datadog.android.rum.integration.tests.elmyr.RumIntegrationForgeConfigurator
+import com.datadog.android.rum.integration.tests.utils.MainLooperTestConfiguration
+import com.datadog.android.tests.assertj.StubEventsAssert
+import com.datadog.tools.unit.annotations.TestConfigurationsProvider
+import com.datadog.tools.unit.extensions.TestConfigurationExtension
+import com.datadog.tools.unit.extensions.config.TestConfiguration
+import fr.xgouchet.elmyr.Forge
+import fr.xgouchet.elmyr.annotation.Forgery
+import fr.xgouchet.elmyr.annotation.IntForgery
+import fr.xgouchet.elmyr.annotation.LongForgery
+import fr.xgouchet.elmyr.annotation.StringForgery
+import fr.xgouchet.elmyr.junit5.ForgeConfiguration
+import fr.xgouchet.elmyr.junit5.ForgeExtension
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.RepeatedTest
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.Extensions
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.junit.jupiter.MockitoSettings
+import org.mockito.quality.Strictness
+import java.net.URL
+
+@Extensions(
+    ExtendWith(MockitoExtension::class),
+    ExtendWith(ForgeExtension::class),
+    ExtendWith(TestConfigurationExtension::class)
+)
+@ForgeConfiguration(RumIntegrationForgeConfigurator::class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+class ManualTrackingRumTest {
+
+    private lateinit var stubSdkCore: StubSDKCore
+
+    @StringForgery
+    private lateinit var fakeApplicationId: String
+
+    @BeforeEach
+    fun `set up`(forge: Forge) {
+        stubSdkCore = StubSDKCore(forge)
+
+        val fakeRumConfiguration = RumConfiguration.Builder(fakeApplicationId)
+            .trackNonFatalAnrs(false)
+            .build()
+        Rum.enable(fakeRumConfiguration, stubSdkCore)
+    }
+
+    // region Feature Init
+
+    @RepeatedTest(16)
+    fun `M create a GlobalRumMonitor W Rum#enable()`() {
+        // Given
+
+        // When
+        val result = GlobalRumMonitor.get(stubSdkCore)
+
+        // Then
+        // We use reflexion because that class is marked internal
```

Comment:
```suggestion
        // We use reflection because that class is marked internal
```

although `reflexion` can also exist in English - `archaic spelling of reflection` 😄

Reply:
Yeah I always struggle between the french and english spellings :/